src/Libraries/CoreNodeModelsWpf/NodeViewCustomizations/CustomSelection.cs

```diff
@@ -27,12 +30,44 @@ public void CustomizeView(CustomSelection model, NodeView nodeView)
             base.CustomizeView(model, nodeView);
 
             var dropdown = (ComboBox)nodeView.inputGrid.Children[1];
+            dropdown.MaxWidth = formControl.Width - leftMargin;
 
             formControl.BaseComboBox = dropdown;
+            formControl.BaseComboBox.SelectionChanged += BaseComboBox_SelectionChanged;
```

Comment:
unsubscribe somewhere?

Reply:
For unsubscribing the method I added the Dispose() method but not sure if this class is being disposed correctly or not.
It's supposed that if you derive from IDispose and implement the Dispose() method the right way to dispose the object should be by using the "`using(instance)`" or by calling  `object.Dispose() `directly.
commit: https://github.com/DynamoDS/Dynamo/pull/14960/commits/c08ab0e2f61b5605c254fd987dc8d385272623ea

Reply:
@QilongTang you were right, when closing the workspace the Dispose method is being reached.
![image](https://github.com/DynamoDS/Dynamo/assets/61755417/a579f4bd-4718-4d8d-9db0-8206dada5822)